tests/unit/test_dataframe.py

```diff
@@ -292,3 +293,14 @@ def test_dataFrame_printSchema(capfd):
         out
         == "root\n |-- A: IntegerType() (nullable = False)\n |-- B: StringType() (nullable = True)\n"
     )
+
+
+def test_session():
```

Comment:
Will you add the mock for `session.connection`?

Reply:
Yes, I just figure out how to make a mock unit test, will add all of them today